crates/consensus/src/receipt/receipts.rs

```diff
@@ -100,6 +101,45 @@ impl<T> From<ReceiptWithBloom<T>> for Receipt<T> {
     }
 }
 
+/// Receipt containing result of transaction execution.
+#[derive(
+    Clone, Debug, PartialEq, Eq, Default, From, derive_more::Deref, DerefMut, IntoIterator,
+)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct Receipts<T> {
+    /// A two-dimensional vector of [`Receipt`] instances.
+    pub receipt_vec: Vec<Vec<Receipt<T>>>,
```

Comment:
actually, let's do:

```suggestion
    pub receipt_vec: Vec<Vec<T>>,
```

this way this is also reusable by multiple things